.github/BOTMETA.yml

```diff
@@ -1394,7 +1394,7 @@ macros:
   team_ansible_core:
   team_aix: MorrisA bcoca d-little flynn1973 gforster kairoaraujo marvin-sinister mator molekuul ramooncamacho wtcross
   team_bsd: JoergFiedler MacLemon bcoca dch jasperla mekanix opoplawski overhacked tuxillo
-  team_consul: sgargan
+  team_consul: sgargan Hakon
```

Comment:
This volunteers Hakon as a maintainer for the consul modules. @Hakon do you agree to this?

Reply:
@Hakon put in all the hard work here, I wasn't sure about what to do here. 

I'm guessing if we don't get a reply or confirmation, I can revert this change.

Reply:
I marked  @Hakon as inactive maintainer in #6755. 

This PR depends on code in the other PR, so leaving this as draft until the other one gets merged to avoid having to fix merge conflicts.